docs/developer-guide/develop-operator.md

```diff
@@ -453,7 +453,8 @@ images:
 ```bash
 cd bpfman-operator/
```

Comment:
those instructions about `cd bpfman-operator` are pre repo split right ? this need to be removed here and everywhere else I think

Reply:
No. The docs are generic for both repositories, so it's trying to indicate which repo the user needs to be in.

Reply:
what does the change directory here imply ?

Reply:
To be in the bpfman-operator directory (repo) and opposed to the bpfman directory (repo). I could change them all to be something like:
```
cd $SRC_DR/bpfman-operator/
```